### DISCUSSION TITLE
Found a way to view last taken photo without the services provider :D

### DISCUSSION THREAD

**Discussion by ianspy1** [lukaspieper/Gcam-Services-Provider#137] [General]
Hi!
If this is known already, please ignore. But on the search for a new Gcam update I stumbled over this project (really like it btw!).
In my Gcam on grapheneOS (gcam version 8.7.250). I can scroll down in settings. And the 4th option in "Additionally" says "Gallery". Changing this from "default" to "Gallery", enabled me to use the last taken pictures function.

I am very new to GitHub. So please forgive me if this was posted elsewhere. I checked the discussions and haven't found a similar post yet.

**Comment by lukaspieper** [comment]
Hi,

In that case, let me welcome you to GitHub. Glad you like my little project here. 

I think you're using a modded version of Gcam and not the official Google Camera app, because I'm using version 9.4.103... and couldn't find that setting. The modded versions often have additional settings.

So this may not be a universal solution for everyone, especially for those who don't want a "tampered" version.
